pom.xml

```diff
@@ -28,7 +28,7 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci</groupId>
     <artifactId>jenkins</artifactId>
-    <version>1.58</version>
+    <version>1.61-20201201.120825-2</version>
```

Comment:
obviously waiting a merge and release of  https://github.com/jenkinsci/pom/pull/150

Reply:
now released 
```suggestion
    <version>1.61</version>
```